src/ansys/dpf/core/incremental.py

```diff
@@ -0,0 +1,237 @@
+"""
+.. _ref_incremental:
+
+Incremental
+========
+"""
+
+from ansys.dpf import core
+
+from typing import Dict, Any
+
+
+class IncrementalHelper:
+    def __init__(
+        self,
+        start_op: core.Operator,
+        end_op: core.Operator,
+        scoping: core.Scoping,
+        scoping_pin: int = None,
+    ):
+        """
+        Given the first and the last operator of a workflow, as well as the scoping.
+
+        This class can be used to simplify the use of incremental operators, and automatically
+        enable to incrementally evaluate a workflow.
+
+        Under the constraint that the end_op supports incremental evaluation.
+        """
+        # input operator should accept a scoping
+        # last operator should support incremental evaluation
+        #     but it should be permissive too (bad doc/spec whatever)
+        self._start_op = start_op
+        self._end_op = self._map_to_incremental(end_op)
+
+        self._scoping = scoping
+        self._scoping_pin = self._find_scoping_pin(scoping_pin)
+
+    def estimate_size(self, max_bytes: int, _dict_inputs: Dict[int, Any]) -> int:
+        """
+        Estimation is based on the size of the output for one id of the given time_scoping.
+        Thus it will run the operator for only "one" iteration.
```

Comment:
```suggestion
        Estimation is based on the size of the output for one ID of the given time_scoping,
        so it will run the operator for only one iteration.
```